utils/check_copies.py

```diff
@@ -132,22 +133,32 @@ def _should_continue(line: str, indent: str) -> bool:
     return line.startswith(indent) or len(line.strip()) == 0 or re.search(r"^\s*\)(\s*->.*:|:)\s*$", line) is not None
 
 
-def find_code_in_transformers(object_name: str) -> str:
+def find_code_in_transformers(object_name: str, base_path: str = TRANSFORMERS_PATH) -> str:
     """
     Find and return the source code of an object.
 
     Args:
-        object_name (`str`): The name of the object we want the source code of.
+        object_name (`str`):
+            The name of the object we want the source code of.
+        base_path (`str`, *optional*, defaults to `TRANSFORMERS_PATH`):
+            The path to the base folder where files are checked.
 
     Returns:
         `str`: The source code of the object.
     """
     parts = object_name.split(".")
     i = 0
 
+    # Detail: the `Copied from` statement is originally designed to work with the last part of `TRANSFORMERS_PATH`,
+    # (which is `transformers`). The same should be applied for `MODEL_TEST_PATH`. However, its last part is `models`
+    # (to only check and search in it) which is a bit confusing. So we keep the copied statement staring with
+    # `tests.models.` and change it to `tests` here.
+    if base_path == MODEL_TEST_PATH:
+        base_path = "tests"
```

Comment:
This is to make `# Copied from tests.models.` work (instead of `# Copied from models.`) and still keep `MODEL_TEST_PATH = "tests/models"`